### ISSUE TITLE
Cannot read property 'nextSibling' of undefined

### ISSUE THREAD

**Issue by ytanruengsri** [DevExpress/testcafe#1442]
### Are you requesting a feature or reporting a bug?
Reporting a bug

### What is the current behavior?
Always getting the following error

**Errors:**
```
Uncaught Error: Cannot read property 'nextSibling' of undefined
```

### What is the expected behavior?
Test should not fail :)

### How would you reproduce the current behavior (if this is a bug)?
Just run the below code

#### Provide the test code and the tested page URL (if applicable)

Tested page URL: https://www.zalando.de/nike-sportswear-sneaker-low-metallic-silvervarsity-redblack-ni111s08k-d11.html

Test code

```js
fixture `Example page`
    .page `https://www.zalando.de/nike-sportswear-sneaker-low-metallic-silvervarsity-redblack-ni111s08k-d11.html`;

test('verify redirection', async t => {
    await t
        .debug()
        .click('.z-navicat-header_logoLinkWrapper');

    const location = await t.eval(() => window.location);

    await t.expect(location.pathname).eql('damen-home/');
});
```

### Specify your

* operating system: MacOSX Sierra 10.12.4
* testcafe version: 0.14.0
* node.js version: v7.7.4

**Comment by AlexanderMoskovkin** [comment]
Hi,

I've reproduced the problem and we will investigate it.

As I workaround you can run the test with the [skip-js-errors](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-e---skip-js-errors) flag:
`testcafe chrome test.js -e`

**Comment by gpsarkar** [comment]
I got a similar one.
` Uncaught TypeError: Cannot read property 'offsetWidth' of undefined`

Sorry, couldn't share the test page here.

**Comment by AlexanderMoskovkin** [comment]
Hi @gpsarkar,
 I'm not sure that the cause of your problem is the same with the problem reported by @ytanruengsri. There are two ways to check it:
1) Wait for fix for the problem described in the ticket initially
2) Create a simple example that allows to reproduce your problem (it's impossible to check your issue without it).
Meanwhile as a workaround you can try to run test with the [skip-js-errors](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-e---skip-js-errors) flag too.

**Comment by gpsarkar** [comment]
@AlexanderMoskovkin Ok. I will try to create a test page to reproduce it.

**Comment by ytanruengsri** [comment]
@AlexanderMoskovkin @gpsarkar with the workaround I'm able to run tests.

**Comment by AlexanderMoskovkin** [comment]
@ytanruengsri Cool! You can use TestCafe with it for now. Meanwhile we are working on the problem and we be able to provide the version with the fix as soon as it's done.

**Comment by LavrovArtem** [comment]
fixed in v0.17.0-alpha4
